scripts/cnn_variant_cromwell_tests/run_cnn_variant_wdl.sh

```diff
@@ -14,12 +14,12 @@ echo "Building docker without running unit tests... ========="
 cd $WORKING_DIR/gatk
 
 # IMPORTANT: This code is duplicated in the cnv and M2 WDL test.
-if [ ${TRAVIS_PULL_REQUEST} != false ]; then
```

Comment:
Can we remove the whole build docker section from all of these and instead use the docker we just built?

Reply:
i don't own these tests and tried to adapt them with the lightest touch i could. I will make an issue but that WILL complicate the fork pr use case we discussed